lib/smart_answer/calculators/business_coronavirus_support_finder_calculator.rb

```diff
@@ -14,8 +14,8 @@ class BusinessCoronavirusSupportFinderCalculator
       job_retention_scheme: ->(calculator) {
         calculator.paye_scheme == "yes"
       },
-      vat_scheme: ->(calculator) {
-        calculator.annual_turnover != "under_85k"
+      vat_scheme: ->(_calculator) {
```

Comment:
We could just remove this entirely: there's a precedent for that [here](https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer_flows/business-coronavirus-support-finder/outcomes/results.govspeak.erb#L180).